examples/asrc_demo/src/i2s_audio.c

```diff
@@ -103,7 +102,7 @@ static void i2s_audio_recv_task(void *args)
 #if PROFILE_ASRC
     uint32_t max_time = 0;
 #endif
-    uint32_t nominal_fs_ratio = 0;
+    uint64_t nominal_fs_ratio = 0;
```

Comment:
Just wondering if this shouldn't be set to the nominal ratio (1:4) initially, or doesn't it matter since it is not acted upon until an I2S frame is rx'd

Reply:
This gets set at run time based on the i2s frequency.